front-end/.env

```diff
@@ -0,0 +1,6 @@
+REACT_APP_AUTH0_DOMAIN = dev-l5smjk3ej12oyjv8.us.auth0.com
```

Comment:
`.env` file should be added in, `.gitignore` & removed from here. It contains sensitive details that should not be available here.